va/va_dec_hevc.h

```diff
@@ -327,7 +475,75 @@ typedef struct  _VASliceParameterBufferHEVC
     uint32_t                va_reserved[VA_PADDING_LOW];
 } VASliceParameterBufferHEVC;
 
+/**
+ * \brief HEVC Extented Slice Parameter Buffer Structure For Long Format
+ *
+ * This data structure contains both baseline HEVC profiles (main, main10)
+ * and extension profiles (range extension and screen content).
```

Comment:
Is the structure below available for screen context coding extension as well? I think the suffix 'Rext' stand for Range Extension only.